server/cmd/main.go

```diff
@@ -285,7 +285,13 @@ func main() {
 			rego = *r
 		}
 
-		krh.SeedKeys(viper.GetString("KEYS_PATH"))
+		//adding check for KEYS_PATH
+		keysPath := viper.GetString("KEYS_PATH")
+		if keysPath == "" {
+			keysPath = "../../server/permissions/keys.csv"
+		}
```

Comment:
@pranjalg1331 @Yashsharma1911 @jerensl, let's ensure that there is a MeshKit error to handle the absence of this.

How does this logic affect that of the local and remove provider logic?

Reply:
@pranjalg1331 are you going to finish this or not?